frontend/src/features/mapSideBar/SidebarContents/subforms/IdentificationForm.tsx

```diff
@@ -113,6 +114,15 @@ export const IdentificationForm: React.FC<IIdentificationProps> = ({
         </Col>
       </Row>
       <hr></hr>
+      <ClassificationForm
+        index="0"
+        nameSpace={withNameSpace('classificationId')}
+        fieldLabel="Building Classification"
+        classifications={classifications}
+        title="Strategic Real Estate Classification"
+        toolTip="Placeholder"
```

Comment:
`Placeholder`

Reply:
Just going to talk to Bobbi about this one, the current tooltip is 'See Inventory Policy for classification definitions' but since the definitions will appear right beside I am wondering if we should just provide information that the definitions will appear upon selection